libr/include/r_util/r_json.h

```diff
@@ -0,0 +1,67 @@
+/* radare - LGPL - Copyright 2020 - thestr4ng3r, Yaroslav Stavnichiy */
+
+#ifndef R_JSON_H
+#define R_JSON_H
+
+#include <r_types.h>
+
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+/*
+ * r_json is a JSON parsing API,
+ * heavily based on nxjson by Yaroslav Stavnichiy <yarosla@gmail.com>,
+ * which is available under LGPLv3 or later.
+ *
+ * r_json does NOT format json, it only parses. To format json, see pj.h instead.
+ * It operates in-place, which means the parsed string will be MODIFIED.
+ * This means all string values in RJson point directly into the input string,
+ * removing the need to copy them.
+ *
+ * It also supports both line and block style comments.
+ */
+
+typedef enum r_json_type_t {
+	R_JSON_NULL,
+	R_JSON_OBJECT,  // properties can be found in child nodes
+	R_JSON_ARRAY,   // items can be found in child nodes
+	R_JSON_STRING,  // value can be found in the str_value field
+	R_JSON_INTEGER, // value can be found in the num.u_value/num.s_value fields
+	R_JSON_DOUBLE,  // value can be found in the num.dbl_value field
+	R_JSON_BOOL     // value can be found in the num.u_value field
```

Comment:
Name it BOOLEAN. To be consistent with the rest of the enum

Reply:
Yes, BOOLEAN is also more consistent with Java :-P